cmd/embedded-cluster/restore.go

```diff
@@ -886,370 +882,364 @@ func installAndWaitForRestoredK0sNode(c *cli.Context, applier *addons.Applier) (
 	return cfg, nil
 }
 
-var restoreCommand = &cli.Command{
-	Name:  "restore",
-	Usage: fmt.Sprintf("Restore a %s cluster", binName),
-	Flags: withProxyFlags(withSubnetCIDRFlags(
-		[]cli.Flag{
-			&cli.StringFlag{
-				Name:   "airgap-bundle",
-				Usage:  "Path to the air gap bundle. If set, the restore will complete without internet access.",
-				Hidden: true,
-			},
-			&cli.StringFlag{
-				Name:  "network-interface",
-				Usage: "The network interface to use for the cluster",
-				Value: "",
-			},
-			&cli.BoolFlag{
-				Name:  "no-prompt",
-				Usage: "Disable interactive prompts.",
-				Value: false,
-			},
-			&cli.BoolFlag{
-				Name:  "skip-host-preflights",
-				Usage: "Skip host preflight checks. This is not recommended.",
-				Value: false,
-			},
-			&cli.BoolFlag{
-				Name:   "skip-store-validation",
-				Usage:  "Skip validation of the backup store. This is not recommended.",
-				Value:  false,
-				Hidden: true,
-			},
-			&cli.StringFlag{
-				Name:  "local-artifact-mirror-port",
-				Usage: "Port on which the Local Artifact Mirror will be served. If left empty, the port will be retrieved from the snapshot.",
-				// DefaultText: strconv.Itoa(defaults.LocalArtifactMirrorPort),
-				Hidden: false,
+func restoreCommand() *cli.Command {
+	runtimeConfig := ecv1beta1.GetDefaultRuntimeConfig()
+
+	return &cli.Command{
```

Comment:
i believe there should be a `data-dir` flag for the restore command as well because the cluster is set up before the backup is accessible.

Reply:
`getDataDirFlag()` function is unsed

Reply:
Done